docs-source/content/samples/simple/azure-kubernetes-service/domain-on-pv.md

```diff
@@ -0,0 +1,549 @@
+   ```
+
+   Make sure the status of cluster-1 is `ServersReady` and `Available`.
+   {{%expand "Click here to view the example status." %}}
+   ```yaml
+   Status:
+    Clusters:
+      Cluster Name:      cluster-1
+      Maximum Replicas:  5
+      Minimum Replicas:  1
+      Ready Replicas:    2
+      Replicas:          2
+      Replicas Goal:     2
+    Conditions:
+      Last Transition Time:  2020-07-06T05:39:32.539Z
+      Reason:                ServersReady
+      Status:                True
+      Type:                  Available
+    Replicas:                2
+    Servers:
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    admin-server
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    managed-server1
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  RUNNING
+      Node Name:      aks-nodepool1-11471722-vmss000001
+      Server Name:    managed-server2
+      State:          RUNNING
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server3
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server4
+      Cluster Name:   cluster-1
+      Desired State:  SHUTDOWN
+      Server Name:    managed-server5
+   ```
+   {{% /expand %}}
+
+To deploy a sample application on WLS, you may skip to the section [Deploy sample application](#deploy-sample-application).  The next section includes a script that automates all of the preceding steps.
+
+#### Automation
+
+If you want to automate the above steps of creating AKS cluster and WLS domain, you can use the script `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks.sh`.
+
+The sample script will create a WLS domain home on the AKS cluster, including
+  - creating a new Azure resource group, with a new Azure Storage Account and Azure File Share to allow WebLogic to persist its configuration and data separately from the Kubernetes pods that run WLS workloads.
+  - creating WLS domain home.
+  - generating the domain resource YAML files, which can be used to restart the Kubernetes artifacts of the corresponding domain.
+
+For input values, you can edit `kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/create-domain-on-aks-inputs.yaml` directly, or copy the file and edit your copy. The following values must be specified:
+
+| Name in YAML file | Example value | Notes |
+|-------------------|---------------|-------|
+| `azureServicePrincipalAppId` | `nr086o75-pn59-4782-no5n-nq2op0rsr1q6` | Application ID of your service principal, refer to the application ID in the [Create Service Principal]({{< relref "/samples/simple/azure-kubernetes-service/domain-on-pv#create-a-service-principal-for-aks" >}}) section. |
```

Comment:
principal, refer -> principal; refer (fix globally)
PS In the Notes in this table, the hyperlinks are fine.

Reply:
Will do.